src/chrome/content/rules/Commandlinefu.com.xml

```diff
@@ -0,0 +1,11 @@
+<ruleset name="Commandlinefu.com">
+
+	<target host="www.commandlinefu.com" />
+
+	<rule from="^http://commandlinefu\.com/"
+		to="https://www.commandlinefu.com/" />
```

Comment:
This will never be used because `commandlinefu.com` is not a listed target host.
You can remove this lines anyway since both aliases support https.